autopilot/log.go

```diff
@@ -12,7 +13,7 @@ var log btclog.Logger
 
 // The default amount of logging is none.
 func init() {
-	UseLogger(build.NewSubLogger("ATPL", nil))
+	UseLogger(build.NewSubLogger("ATPL", signal.Interceptor{}, nil))
```

Comment:
this change seems a bit large and unnecessary to me. I get that it is a way to request a shutdown without going through a global var, but I'm hesitant to merge such a large change.

I think it is better to leave the global connection for now, until we have a better understanding of whether this is needed. A way would just keep the latest created struct in the global scope such that it can easily be requested to shut down.

Reply:
@halseth I think the scope has been reduced significantly by @carlaKC's [code improvements](https://github.com/carlaKC/lnd/commit/abfce52064fe5d1e684e81d5a2f7e64fb0ffb56c). So perhaps we can continue with the current path.